.changelog/27fb280ebf754001a8194e392169c032.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id": "27fb280e-bf75-4001-a819-4e392169c032",
+    "type": "bugfix",
+    "description": "Move internal changelog and release tooling into shared repository.",
+    "modules": [
+        "internal/repotools"
```

Comment:
Technically not required as this module is marked with `no_tag`, but this won't cause any issues.